x/mongo/driver/mongocrypt/options/mongocrypt_context_options.go

```diff
@@ -63,3 +73,15 @@ func (eeo *ExplicitEncryptionOptions) SetAlgorithm(algorithm string) *ExplicitEn
 	eeo.Algorithm = algorithm
 	return eeo
 }
+
+// SetQueryType specifies the query type.
+func (eeo *ExplicitEncryptionOptions) SetQueryType(queryType *QueryType) *ExplicitEncryptionOptions {
```

Comment:
Can the param type to be `QueryType` instead of `*QueryType`?

Reply:
Done.